pkg/caimake/cmd/clean.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"github.com/caicloud/nirvana/cli"
+	"github.com/spf13/cobra"
```

Comment:
nirvana should capsulate all underline information, importing `cobra` here worries me.